x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/prebuilt_rules/install_notifications.cy.ts

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { createRuleAssetSavedObject } from '../../../../helpers/rules';
+import {
+  ADD_ELASTIC_RULES_BTN,
+  ADD_ELASTIC_RULES_EMPTY_PROMPT_BTN,
+} from '../../../../screens/alerts_detection_rules';
+import { deleteFirstRule } from '../../../../tasks/alerts_detection_rules';
+import {
+  deleteAlertsAndRules,
+  deletePrebuiltRulesAssets,
+} from '../../../../tasks/api_calls/common';
+import {
+  installAllPrebuiltRulesRequest,
+  installPrebuiltRuleAssets,
+  installMockEmptyPrebuiltRulesPackage,
+  installSpecificPrebuiltRulesRequest,
+} from '../../../../tasks/api_calls/prebuilt_rules';
+import { resetRulesTableState } from '../../../../tasks/common';
+import { login } from '../../../../tasks/login';
+import { visitRulesManagementTable } from '../../../../tasks/rules_management';
+
+describe(
+  'Detection rules, Prebuilt Rules Installation and Update Notifications',
+  { tags: ['@ess', '@serverless', '@skipInServerlessMKI'] },
+  () => {
+    before(() => {
+      installMockEmptyPrebuiltRulesPackage();
+    });
+
+    beforeEach(() => {
+      deleteAlertsAndRules();
+      deletePrebuiltRulesAssets();
+      /* Make sure persisted rules table state is cleared */
+      resetRulesTableState();
+
+      login();
+    });
+
+    describe('No notifications', () => {
+      it('does NOT display install notifications when no prebuilt assets and no rules are installed', () => {
+        visitRulesManagementTable();
+
+        cy.get(ADD_ELASTIC_RULES_EMPTY_PROMPT_BTN).should('be.visible');
+        cy.get(ADD_ELASTIC_RULES_BTN).should('have.text', 'Add Elastic rules');
+      });
+
+      // https://github.com/elastic/kibana/issues/179967
+      it(
+        'does NOT display install notifications when latest rules are installed',
+        { tags: ['@skipInServerlessMKI'] },
+        () => {
```

Comment:
It looks like these tests were just copied over from the old `notifications.cy.ts` file but maybe worth cleaning up some of these old comments and tags. This whole suite has the `@skipInServerlessMKI` tag, probably don't need it again here. Some of the tickets commented are also closed as not planned, not sure if they're worth keeping for context or not

Reply:
Thanks for noticing that 👍 

Due to time constraints I had to move the tests almost with out the changes. We should approach prebuilt rules installation test plan implementation comprehensively where we could clean all this up.